build-tests/api-extractor-test-01/src/index.ts

```diff
@@ -42,8 +42,8 @@ export class AmbientConsumer {
    * Configured via tsconfig.json's "lib" setting, which specifies `@types/jest`.
    * The emitted index.d.ts gets a reference like this:  <reference types="jest" />
    */
-  public definitelyTyped(): jest.Context {
-    return {} as jest.Context;
+  public definitelyTyped(): object {
```

Comment:
@octogonz  Context is not exported from 'jest' anymore, it is now exported in `@jest/reporters`.

Reply:
@sachinjoseph This test doesn't actually care about Jest.  It just needs some example of a globally defined symbol.  I chose to use Jest because it's already imported and all their test APIs are global symbols.

Reply:
Aha! I kind of suspected that but couldn't be certain, thank you for confirming. I will update test to use some other global symbol from Jest.

Reply:
I pushed an update that changes `jest.Context` to `jest.EmptyFunction`

Reply:
Thanks!

Reply:
Changed this to jest.MockContext to uncomment the test.